_format/1.1/index.md

```diff
@@ -475,12 +475,15 @@ as attributes. Instead, [relationships] **SHOULD** be used.
 ##### <a href="#document-resource-object-relationships" id="document-resource-object-relationships" class="headerlink"></a> Relationships
 
 The value of the `relationships` key **MUST** be an object (a "relationships
-object"). Members of the relationships object ("relationships") represent
-references from the [resource object][resource objects] in which it's defined to other resource
-objects.
+object"). Each member of the relationships object ("relationships") represents
+a "relationship", i.e., a reference from the [resource object][resource objects]
+in which they are defined to other resource objects.
```

Comment:
I'd like to try to define this section a bit more crisply. What do you think of this?

```suggestion
object"). Each member of a relationships object ("relationships") represents
a "relationship" from the [resource object][resource objects]
in which it has been defined to other resource objects.
```

Reply:
Thanks a lot. Reads much better.